pyscf/lib/libqcschema.py

```diff
@@ -0,0 +1,220 @@
+import json
+import numpy as np
+from pyscf.lib.parameters import BOHR
+
+def load_qcschema_json( file_name ):
+    # load qcschema output json file
+    data = None
+    with open(file_name,'r') as f:
+        data = json.load(f)
+    return data
+
+def load_qcschema_go_final_json( file_name ):
+    # load qcschema GO output json file 
+    # and return last 'trajectory' point's entries
+    # (this is the optimized molecule)
+    data = None
+    temp = None
+    with open(file_name,'r') as f:
+        temp = json.load(f)
+    data = temp["trajectory"][-1]
+    return data
+
+def combine_syms_coords(syms,coords,to_Angstrom = False, xyz=False):
```

Comment:
This API seems not necessary. code in this function can be integrated into `load_qcschema_molecule`